### PR TITLE
Allow file banner comments to be documentation comments

### PR DESCRIPTION
**Pull request by JoeRobich**
This PR fixes a bad merge from master to master-vs-deps.

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs [hunk 1]
```diff
@@ -114,6 +114,112 @@ class Program2
...
+{
+    static void Main()
+    {
+    }
+}
+        </Document>
+        <Document>/// This is the banner
+/// It goes over multiple lines
+
+class Program2
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs]: this case worries me.  from the perspective of hte language, isn't this a comment on the class?

**Comment by CyrusNajmabadi** [comment]
Question: why are we doing this?  The original work here explicitly carved these out intentionally.  I'm curious why the thinking here changed.  Can you clarify what's driving this?  Thanks!

File: src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs [hunk 2]
```diff
@@ -114,6 +114,112 @@ class Program2
...
+{
+}
+        </Document>
+    </Project>
+</Workspace>",
+@"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <Document>/// This is the banner
+/// It goes over multiple lines
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs]: doesn't this produce a warning about a doc comment on an invalid element... i'm not sure we should treat this as a file banner.

**Comment by JoeRobich** [comment]
@CyrusNajmabadi This simply moves a change from master to master-vs-deps that was missing due to a bad merge conflict resolution.

**Comment by CyrusNajmabadi** [comment]
> @CyrusNajmabadi This simply moves a change from master to master-vs-deps that was missing due to a bad merge conflict resolution.

Ah gotcha.  I must have missed the original PR.  Do you know why it was done?

**Comment by JoeRobich** [comment]
Here is the original PR https://github.com/dotnet/roslyn/pull/33258

**Comment by sharwell** [comment]
Yep @CyrusNajmabadi This is a cherry pick to fix a broken merge. Please file feedback items as issues.

**Comment by CyrusNajmabadi** [comment]
Sounds good.  Looks like i also can't remember things from 5 weeks ago .  SMH.

**Pull request merged**